Tensile/ClientWriter.py

```diff
@@ -95,6 +97,29 @@ def main( config ):
   functions = []
   functionNames = []
   enableHalf = False
+
+  # Note: this step is needed for retrieving code object and tensile library yaml paths
+  # TODO Do away with this workaround. TensileCreateLibrary.py should take function arguments and return values directly
+  #      Refactor TensileCreateLibrary (not .py) to take CLI args instead and pass to underlying python script
+  globalParameters["GenerateManifestAndExit"] = True
+  createLibraryScript = getBuildNewClientLibraryScript(stepBaseDir, libraryLogicPath)
+  subprocess.run(shlex.split(createLibraryScript), cwd=stepBaseDir)
+  coList = []
+  yamlList = []
+  with open(os.path.join(stepBaseDir,"library","TensileManifest.txt"), "r") as f:
```

Comment:
This should be configurable.

Reply:
You mean the TensileCreateLibrary input args? The integration test is already parametrized by a few of the available options of the library creation script.

Reply:
I am mistaken—you were actually referring to the ClientWriter. Could you be more specific about what should be configurable?

Reply:
I mean the location of the manifest file.

Reply:
The relative path to the manifest file (`library/TensileManifest.txt`) is not changeable from user's perspective. Same goes to TensileConfig.cmake, which has the exact hardcoded path. Or if you meant the parent folder `stepBaseDir`, in which case it is already configurable via `globalParameters["LibraryClientPath"]`

Reply:
@sdquiring ping. further comment is appreciated.

Reply:
I suppose making this fully configurable is outside the scope of this PR.